spec/index.html

```diff
@@ -547,21 +548,45 @@ <h2>RDF Graphs</h2>
 
   <p>An <dfn>RDF graph</dfn> is a set of <a>RDF triples</a>.</p>
 
-  <p>An RDF triple is said to be <dfn id="dfn-asserted-triple" data-lt="asserted|asserted triple">asserted</dfn> in an RDF graph if it is an element of the RDF graph.</p>
+  <p>An [=RDF triple=] is said to be <dfn id="dfn-asserted-triple" class=export data-lt="asserted triple|asserted triples" data-lt-no-plural=""
+      >asserted</dfn> in an [=RDF graph=] if it is an element of the [=RDF graph=].
+  </p>
 
   <section id="section-triples">
     <h3>Triples</h3>
 
-    <p>An <dfn data-local-lt="triple">RDF triple</dfn> (usually called "triple")
-      is a 3-tuple (|s|, |p|, |o|) with the following characteristics:</p>
+    <p>An <dfn data-local-lt="triple" class="export">RDF triple</dfn>
+     (often simply called "triple")
+     is a 3-tuple that is defined inductively as follows:</p>
 
     <ul>
-      <li>|s| is an <a>IRI</a> or a <a>blank node</a>.</li>
-      <li>|p| is an <a>IRI</a>.</li>
-      <li>|o| is an <a>IRI</a>, a <a>blank node</a>, a <a>literal</a>,
-        or a <a>triple term</a>.</li>
+      <li>
+        If |s| is an [=IRI=] or a [=blank node=],
+        |p| is an [=IRI=], and
+        |o| is an [=IRI=], a [=blank node=], or a [=literal=],
+        then (|s|, |p|, |o|) is an [=RDF triple=].
+      </li>
+
+      <li>
+        If |s| is an [=IRI=] or a [=blank node=],
+        |p| is an [=IRI=], and
+        |o| is an [=RDF triple=],
+        then (|s|, |p|, |o|) is an [=RDF triple=].
+      </li>
     </ul>
 
+    <p>The three components (|s|, |p|, |o|) of an [=RDF triple=] are respectively called the <dfn class=export >subject</dfn>, <dfn class=export >predicate</dfn> and <dfn class=export >object</dfn> of the triple.</p>
+
+
+    <p class="note">The definition of <a>triple</a> is recursive.
+      That is, a <a>triple</a> can itself have an
+      <a>object</a> component which is another <a>triple</a>.
+      However, by this definition, cycles of <a>triples</a> cannot be created.</p>
+  </section>
+
+  <section id="section-terms">
+    <h3>RDF terms</h3>
```

Comment:
```suggestion
    <h3>RDF Terms</h3>
```
To be consistent with the way the other section titles are capitalized.